programs/vote/src/vote_state/mod.rs

```diff
@@ -141,6 +141,37 @@ pub fn to<T: WritableAccount>(versioned: &VoteStateVersions, account: &mut T) ->
     VoteState::serialize(versioned, account.data_as_mut_slice()).ok()
 }
 
+// Updates the vote account state with a new VoteState instance.  This is required temporarily during the
+// upgrade of vote account state from V1_14_11 to Current.
+fn set_vote_account_state(
```

Comment:
lets add a test for this to verify the correct format is being stored and properly read back with `vote_account.get_state::<VoteStateVersions>()`

Reply:
Yes, I was planning to write tests like that, but was waiting to make sure that this change's overall shape is acceptable before proceeding with that effort.  Since I haven't heard any push back about the overall shape of the change, I'll add the tests.  Thanks!